data_model/src/query/mod.rs

```diff
@@ -40,6 +41,50 @@ pub mod cursor;
 pub mod pagination;
 pub mod sorting;
 
+const FETCH_SIZE: &str = "fetch_size";
+
+/// Default value for `fetch_size` parameter in queries.
+#[cfg(feature = "std")]
+pub static DEFAULT_FETCH_SIZE: once_cell::sync::Lazy<NonZeroU32> =
```

Comment:
Maybe instead of `once_cell` we can use [small function](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=31efe0ae459721e7df303b92fc7f997c) with a little bit of unsafe? Safety is checked at compile time.

Upside is that it can be used in `std` and `no_std` environment without problem and require lesser amount of code.

Reply:
Nice idea, will do!

Reply:
I think just in-place `unsafe` block would be fine here as well, check out new code, please